http-link-header.cabal

```diff
@@ -1,73 +1,98 @@
-name:            http-link-header
-version:         1.1.0
-synopsis:        A parser and writer for the HTTP Link header as specified in RFC 5988 "Web Linking".
-description:     https://github.com/myfreeweb/http-link-header
-category:        Web
-homepage:        https://github.com/myfreeweb/http-link-header
-author:          Greg V
-copyright:       2014-2020 Greg V <greg@unrelenting.technology>
-maintainer:      stevenjshuck@gmail.com
-license:         PublicDomain
-license-file:    UNLICENSE
-build-type:      Simple
-cabal-version:   >= 1.10
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.33.0.
```

Comment:
Probably better to not have both files at the same time in the repo, only the yaml